conductor_api/src/interface.rs

```diff
@@ -138,8 +138,18 @@ impl ConductorApiBuilder {
             let hc_lock = instance.clone();
             let hc_lock_inner = hc_lock.clone();
             let mut hc = hc_lock_inner.write().unwrap();
-            let call_params = params_map.get("params");
-            let params_string = serde_json::to_string(&call_params)
+
+            // Getting the arguments of the call contained in the json-rpc 'params'
+            let call_args = match params_map.get("args") {
```

Comment:
I think you can do away with the match like this:

```rust
let call_args = params_map.get("args").or_else(|| {
    // do the log
    params_map.get("params")
});
```

Reply:
This is actually cleaner code thanks! :+1: